Submodels/MTFLO_call.py

```
@@ -1,20 +1,183 @@
+
+        # Generate the subprocess and write it to self
+        self.process = subprocess.Popen([self.fpath, self.analysis_name], 
+                                        stdin=subprocess.PIPE, 
+                                        stdout=subprocess.PIPE, 
+                                        stderr=subprocess.PIPE,
+                                        shell=True, 
+                                        text=True,
+                                        )
+        
+        # Check if subprocess is started successfully
+        if self.process.poll() != None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)